src/Model/Resolver/Product/PriceRange.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * ScandiPWA_CatalogGraphQl
+ *
+ * @category    ScandiPWA
+ * @package     ScandiPWA_CatalogGraphQl
+ * @author      Yefim Butrameev <info@scandiweb.com>
+ * @copyright   Copyright (c) 2020 Scandiweb, Ltd (https://scandiweb.com)
+ */
+
+declare(strict_types=1);
+
+namespace ScandiPWA\CatalogGraphQl\Model\Resolver\Product;
+
+use Magento\Catalog\Model\Product;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Query\Resolver\ContextInterface;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\Store\Api\Data\StoreInterface;
+use Magento\CatalogGraphQl\Model\Resolver\Product\Price\Discount;
+use Magento\CatalogGraphQl\Model\Resolver\Product\Price\ProviderPool as PriceProviderPool;
+use Magento\CatalogGraphQl\Model\Resolver\Product\PriceRange as CorePrice;
+
+/**
+ * Format a product's price information to conform to GraphQL schema representation
+ */
+class PriceRange extends CorePrice
+{
+    /**
+     * @var PriceProviderPool
+     */
+    private $priceProviderPool;
+
+    /**
+     * @var Discount
+     */
+    private $discount;
+
+    /**
+     * @param PriceProviderPool $priceProviderPool
+     * @param Discount $discount
+     */
+    public function __construct(
+        PriceProviderPool $priceProviderPool,
+        Discount $discount
+    ) {
+        $this->priceProviderPool = $priceProviderPool;
+        $this->discount = $discount;
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * Format product's tier price data to conform to GraphQL schema
+     *
+     * @param \Magento\Framework\GraphQl\Config\Element\Field $field
+     * @param ContextInterface $context
+     * @param ResolveInfo $info
+     * @param array|null $value
+     * @param array|null $args
+     * @throws \Exception
+     * @return array
+     */
+    public function resolve(
+        Field $field,
+        $context,
+        ResolveInfo $info,
+        array $value = null,
+        array $args = null
+    ) {
+        if (!isset($value['model'])) {
+            throw new LocalizedException(__('"model" value should be specified'));
+        }
+        /** @var StoreInterface $store */
+        $store = $context->getExtensionAttributes()->getStore();
+
+        /** @var Product $product */
+        $product = $value['model'];
```

Comment:
Original class has the `$product->unsetData('minimal_price');` here, why is that removed?